docs/benchmarks/data-comparisson.md

```diff
@@ -64,6 +64,10 @@ CLASSIFICATIONS. #
 
 **Other files:**<br />The `lisa.rel`file is an old version of the relevance assessments which helps to understand new version, but is not considered important to use. 
 
+**Parsing problems**:<br />
+* documents: they are all in seperate files, the newline as online marker to seperate ID, title and text can be a bit tricky
+* relevance assessments: not parseable line by line, the number of relevant documents should be used to parse
```

Comment:
Sentence seems cut off, missing some information

Reply:
I only added them as bullet points so I chose short sentences but the information is all in this sentence, should I change it to a normal sentence?